### PR TITLE
Update comments to reduce confusion

### PR DESCRIPTION
**Pull request by cici37**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This pr is to update comments to reduce confusion

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/cloud-provider/app/controllermanager.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ const (
 )
 
 // NewCloudControllerManagerCommand creates a *cobra.Command object with default parameters
-// initFuncConstructor is a map of named controller groups (you can start more than one in an init func) paired to their InitFuncConstructor.
+// initFuncConstructor is a map of named controller groups paired to their InitFuncConstructor.
```
Comment (cheftako) [inline staging/src/k8s.io/cloud-provider/app/controllermanager.go:69]: "map of name controller groups to ... InitFuncConstructor" makes me thinks the key is either a group of controllers or a name for a group of controllers. Might be better to indicate a key is the external controller name (as defined by controllers flag in https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#options or the CCM equivalent)

**Comment by nckturner** [comment]
Definitely an improvement.  
/lgtm

**Comment by cici37** [comment]
@cheftako for approval(rebased on the pow job failure fix)

**Comment by cheftako** [comment]
Comment only fix. Significantly reduces confusion.

**Pull request merged**